iceoryx_utils/include/iceoryx_utils/posix_wrapper/timer.hpp

```diff
@@ -321,6 +321,58 @@ class Timer
 };
 
 } // namespace posix
+
+
+namespace clock
```

Comment:
Wrong namespace. The namespace should be `posix`

Reply:
Since this doesnt have the POSIX abstraction this was removed. The new name space planned is given in the the other comment. Kindly verify.